src/oas2/__tests__/operation.test.ts

```diff
@@ -226,7 +229,7 @@ describe('transformOas2Operation', () => {
   });
 
   it('given malformed parameters should translate operation with those parameters', () => {
-    const document: Partial<OpenAPIObject> = {
+    const document: Partial<DeepPartial<Spec>> = {
```

Comment:
Does this need the `Partial`? Wouldn't this be the same as just `DeepPartial<Spec>`?

Reply:
i can fix that